docs/developers/performance-test-setup-for-karmada.md

```diff
@@ -0,0 +1,331 @@
+        - key: "fake-kubelet/provider"
+          operator: "Exists"
+          effect: "NoSchedule"
+      # nodeName: fake-0 # Or direct scheduling to a fake node
+      containers:
+        - name: fake-pod
+          image: fake
+EOF
+```
+
+`kubectl get pod` You will find that it has been started, although the image does not exist.
+
+```shell
+> kubectl get pod -o wide
+NAME                        READY   STATUS    RESTARTS   AGE   IP          NODE     NOMINATED NODE   READINESS GATES
+fake-pod-78884479b7-52qcx   1/1     Running   0          6s    10.0.0.23   fake-4   <none>           <none>
+fake-pod-78884479b7-bd6nk   1/1     Running   0          6s    10.0.0.13   fake-2   <none>           <none>
+fake-pod-78884479b7-dqjtn   1/1     Running   0          6s    10.0.0.15   fake-2   <none>           <none>
+fake-pod-78884479b7-h2fv6   1/1     Running   0          6s    10.0.0.31   fake-0   <none>           <none>
+```
+
+
+
+## Distribute resources using CLusterLoader2
+
+### ClusterLoader2
+
+[ClusterLoader2](https://github.com/kubernetes/perf-tests/tree/master/clusterloader2) is an open source Kubernetes cluster testing tool. It tests against Kubernetes-defined SLIs/SLOs metrics to verify that clusters meet various quality of service standards. ClusterLoader2 is a tool oriented single cluster, it is complex to test karmada control plane meanwhile distribute resouces to member clusters. Therefore, we just use the ClusterLoader2 to distribute resources to clusters managed by karmada.
+
+### Prepare a simple config
+
+Let's prepare our config (config.yaml) to distribute resources. This config will:
+
+- Create 10 namespace
+
+- Create 20 deployments with 1000 pods inside that namespace
+
+
+We will create file `config.yaml` that describes this test. First we need to start with defining test name:
+
+```yaml
+name: test
+```
+
+ClusterLoader2 will create namespaces automatically, but we need to specify how many namespaces we want:
+
+```yaml
+namespace:
+  number: 10
+```
+
+Next, we need to specify TuningSets. TuningSet describes how actions are executed. The qps means 1/qps s per action interval. In order to distribute resources slowly to relieve the pressure on the apiserver, the qps of Uniformtinyqps is set to 0.01, which means that after distributing a deployment, we wait 10s before continuing to distribute the next deployment.
+
+```yaml
+tuningSets:
+- name: Uniformtinyqps
+    qpsLoad:
+    qps: 0.01
+- name: Uniform1qps
+  qpsLoad:
+    qps: 1
+```
+
+Finally, we will create a phase that creates deployment and propagation policy . We need to specify in which namespaces we want the deployment and propagation policy to be created, how many of these deployments per namespace. Also, we will need to specify template for our deployment and propagation policy , which we will do later. For now, let's assume that this template allows us to specify number of replicas in deployment.
```

Comment:
specify in which namespaces we want the deployment and propagation policy to be created and how many of these deployments per namespace. Also, we will need to specify template for our deployment and propagation policy,